### PR TITLE
Add Input Component

### PR DESCRIPTION
**Pull request by manojkumar8400**
Add Input Component

1. Text-box Input
2. Validation and errors style
3. Text-area Input
4. Radio Input
5. Checkbox Input

### CODE REVIEW

File: documentation/input/input.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@import "/documentation/Styles/general.css";
+
+#input-compo {
+    background-color: var(--header);
+    border-radius: 25px;
+}
+
+.code-cntnr {
```
Comment (mimansha-swarup) [inline documentation/input/input.css:8]: use proper name like center instead of cntnr

**Review by mimansha-swarup — COMMENTED** [review]
rest looks good to me

File: documentation/input/input.css (hunk 1 above)
Comment (manojkumar8400) [reply documentation/input/input.css:8]: Ok
but It's a short form of container.

Comment (msdeshmukh009) [reply documentation/input/input.css:8]: Hey, Manoj it's actually confusing to me as well. I also thought its `.code-center`

**Review by msdeshmukh009 — APPROVED** [review]

**Comment by manojkumar8400** [comment]
Ok, I will change it.

**Pull request merged**